File: bingo number genertor.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <conio.h>
#include <time.h>
main(){
    system("COLOR 3A");
	char ans;
	char Bingo[] ={'B','I','N','G','O'};
	int a;
	srand(time(NULL));
	printf("Welcome to Bingo would you wish to play? y/n\n");
	fflush(stdin);
	ans=getchar();
	while(ans=='y'){
		a=rand()%99+1;
		printf("The number is %c%d\n", Bingo[rand()%5],a);
		printf("Continue calling numbers? y/n\n");
		fflush(stdin);
		ans=getchar();
		if (ans !='y'){
            system("COLOR 9B");
			printf("BINGO!!!!!!!!!!!!!!!!!!!!!!");
			printf("We hope you enjoy your prize");
			break;
		}
	}
}

File: bingo number genertor.js
//Bingo Number Generator
/*
first attempt at codeing after bingeing js and html/css tutorials.... havent figured out how to test yet 
To figure out how to test later so that i can make this look pretty :s 
*/

window.onload=function (){
	var ans = confirm("Welcome to Bingo would you wish to play?");
	var Bingo =['B','I','N','G','O'];
	while(ans){
		var a =Bingo[Math.floor((Math.random()*5))]+Math.floor((Math.random()*99)+1);
		ans = confirm("Continue naming");
		if (!ans){
			console.log("BINGO!!!!!!!!!!!!!!!!!!!!!!");
			console.log("We hope you enjoy your prize");
			break;
		}
	}
document.getElementById("calendar-month-year").innerHTML = a;
}